Cargo.toml

```diff
@@ -39,4 +39,4 @@ members = [
 
 [patch.crates-io]
 ocaml-boxroot-sys = { git = "https://gitlab.com/bruno.deferrari/ocaml-boxroot.git", branch = "ocaml-410-headers" }
-librocksdb-sys = { git = "https://github.com/tezedge/rust-rocksdb.git", tag = "v1.5.0-tezedge-compatible" }
+librocksdb-sys = { git = "https://github.com/tezedge/rust-rocksdb.git", tag = "tezedge-v1.0.0" }
```

Comment:
@binier @vlad9486 
I checked that fork, but this tag is not based on rocksdb `v0.17.0`, why not?
but `origin/tezedge` is based on rocksdb `v0.17.0`
shouldnt be all above tag `v0.17.0`??

and I would change our tag from `tezedge-v1.0.0` to `tezedge-v0.17.0`, it will be much clear



![image](https://user-images.githubusercontent.com/8159517/127652907-ea12b180-e37d-498e-9236-479bcd8d673b.png)

Reply:
![image](https://user-images.githubusercontent.com/16461497/127653585-ca629498-62d5-4cbf-9ffa-d12ffa9209a7.png)
It's 1 commit after release of 0.17, just to add enable frame pointers.

About the tag, I just wanted the tag to not be based on anything, so that if we want to change something in rocksdb.

If we don't do it this way, there are two options:
- Have just one tag: `tezedge-v0.17.0`, and when changing **rust-rocksdb** in any way, we simply move that tag to newer commit. That will be problematic since when moving the tag, older releases might break.
- We could change version's patch number, but then if **rust-rocksdb** releases a patch, that will make version misleading.

Another option and probably best is to add revision number or something like that to tag, and on change we increment that `tezedge-v0.17.0-1`, `tezedge-v0.17.0-2`, etc...

Reply:
@binier 
very well, I like your last option, `tezedge-v0.17.0-1`,
we will exactly know which version of rocksdb it is and also, which is our fix/patch version

Reply:
@binier 
ignore that rebase studd, now I see it, my repo was not updated correctly
```
 ! [rejected]        tezedge-v1.0.0 -> tezedge-v1.0.0  (would clobber existing tag)
```